controller/src/main/java/cern/jarrace/controller/domain/Service.java

```diff
@@ -0,0 +1,67 @@
+package cern.jarrace.controller.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author timartin
+ * Domain class that represents a registered {@link Service}
+ */
+
+public class Service {
+
+    /**
+     * Name of the agent that can execute the service
+     */
+    String agentName;
+
+    /**
+     * Class used by the exposed {@link Service}
+     */
+    String clazz;
+
+    /**
+     * {@link List} of entrypoints exposed by this service in the specific class
+     */
+    List<String> entrypoints;
+
+    public Service() {
+        entrypoints = new ArrayList<>();
+    }
+
+    public Service(String agentName, String clazz) {
+        this.agentName = agentName;
+        this.clazz = clazz;
+        entrypoints = new ArrayList<>();
+    }
+
+    public Service(String agentName, String clazz, List<String> entrypoints) {
+        this.agentName = agentName;
+        this.clazz = clazz;
```

Comment:
no not use miss Spelled names! find a better one: What class is it? further its a className
Could it be a ''entyPointClassName"?